priv/repo/migrations/20190206165445_move_enums_into_ret0.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Ret.Repo.Migrations.MoveEnumsIntoRet0 do
+  use Ecto.Migration
+
+  def up do
+    execute("alter type public.scene_state set schema ret0")
```

Comment:
ah good catch, this is because the migration of enums wrt schema is dictated by the code at HEAD (since it dispatches to the enum definitions to find the schema) -- so if you replay the migrations it will end up being in the right schema the first time through now since the schema is now correct at HEAD

Reply:
this is a bit gnarly, what i'm going to do instead is just let this run when i push to fix the problem, and then comment out + commit the migration script. at that point, everything will work as expected and the live schemas will be correct.